README.md

```diff
@@ -2,6 +2,8 @@
 
 This repository contains Dockerfile for PgBouncer for Fabric8-Analytics project.
 
+It also contains a openshift template for [PgWeb](https://github.com/sosedoff/pgweb), which would useful for DB visualization.
```

Comment:
Can you also add where pgweb will come up (http://localhost:8081) or something else?

Reply:
Thanks @miteshvp. Added more info.